src/libs/blueprint/conduit_blueprint_mesh.hpp

```diff
@@ -86,6 +86,70 @@ void CONDUIT_BLUEPRINT_API generate_index(const conduit::Node &mesh,
                                           index_t num_domains,
                                           Node &index_out);
 
+namespace connectivity
```

Comment:
So just put these helpers completely in conduit_blueprint_mpi_mesh.cpp?  That is fine, as I will defer to you on these kinds of code organization matters.  I separated them because they have no MPI, but if their only use is to enable MPI-based functionality, it still makes sense.